### DISCUSSION TITLE
BUG, hack or what? My 2 xch coins are missing

### DISCUSSION THREAD

**Discussion by lucamarce** [Chia-Network/chia-blockchain#6906] [Support]
Hello,
I have a problem to submit to you and I hope you can help me.

I was farmed 2 xch this month (16/06/21), now my wallet show 0 xch

my first address: xch1ythl3t2d637rp6d90eudklqw7lpfuuyz8p4jlu4t27f3fcqsyfzs4zc5e5
the second : xch1rf55rxy8wtgjkszfldtpavhu34dtpkrmx2nzww4a0whljrlzx8ks47d2vq

1,75	Incoming xch1ythl3t2d637rp6d90eudklqw7lpfuuyz8p4jlu4t27f3fcqsyfzs4zc5e5 06/16/2021 01:05:26 Confirmed at height 436952
0,25	Incoming xch1ythl3t2d637rp6d90eudklqw7lpfuuyz8p4jlu4t27f3fcqsyfzs4zc5e5 06/16/2021 01:05:26 Confirmed at height 436952

In chiaexplorer now is in status “spent” on block 439.273 

Why if I don't have outgoing transactions?

I use Chia on a dedicated machine where I have never installed other software except windows, chia (last release) and swar chia plot manager. I have never participated in pools

![01_Chia_Farm](https://user-images.githubusercontent.com/86286902/122882807-d762b180-d33c-11eb-88aa-f068fcdadd49.PNG)

![02_Chia_Hystory](https://user-images.githubusercontent.com/86286902/122882810-d7fb4800-d33c-11eb-97cf-45129611b32a.PNG)

![03_Chia_Wallet](https://user-images.githubusercontent.com/86286902/122882811-d893de80-d33c-11eb-9b34-98bd5bd639e8.PNG)

Thanks for the support
Luca

**Comment by loppefaaret** [comment]
I'm sorry, but if you are not the one transferring the coins, someone else is in possession of your private 24 workd mnemonic!
if this is your plots private key, look into here to securely farm them going forward:
https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Keys-Management

**Reply by lucamarce** [reply]
Very strange. My private 24 workd mnemonic is only on paper and the computer has been installed for a month only for official Chia.

Shouldn't I see the transaction in the History?

**Reply by lucamarce** [reply]
I took your advice and created a cold storage key

Better to format my computer?

Thank you
Luca

**Reply by loppefaaret** [reply]
if you wanna be sure, that is perhaps a good thing to do, and create a new key again afterwards!

**Comment by BrandtH22** [comment]
@lucamarce , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
